crates/sui-types/src/gas.rs

```diff
@@ -381,7 +378,7 @@ impl<'a> SuiGasStatus<'a> {
     }
 
     pub fn charge_vm_gas(&mut self) -> Result<(), ExecutionError> {
-        // Disable flat fee for now
+        // MUSTFIX: Disable flat fee for now. Do we ever enable this?
```

Comment:
This is what we should remove @lxfind 
Re: this [comment](https://github.com/MystenLabs/sui/pull/9282/files#r1136101848)